docs/tutorials/07_pegasos_qsvc.ipynb

```diff
@@ -24,7 +24,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 1,
```

Comment:
The above text was not changed but in going over the PR 

> There's another SVM based algorithm that benefits from the quantum kernel method. Here, we introduce an implementation of a another classification algorithm, which is an alternative version to the QSVC shown above.

this "... which is an alternative version to the QSVC shown above." bit seems out of place. Maybe a link to the other tutorial that shows the QSVC - i.e. the 03_ one

Reply:
Replace with "an alternative version to the `QSVC` available in Qiskit Machine Learning".
I guess it is better to avoid links to the pages that are generated automatically.

Reply:
You can link notebooks - its supported. For instance here in Optimization 02_converters.... it links to another

> `See the following tutorial about the [Minimum Eigen Optimizer](./03_minimum_eigen_optimizer.ipynb) for more detail`

You can see it [live here](https://qiskit.org/documentation/optimization/tutorials/02_converters_for_quadratic_programs.html) at the end of the 2nd intro paragraph where its been converted now to html. But it works too in the original notebook if you [view it directly on the optimization repo](https://github.com/Qiskit/qiskit-optimization/blob/main/docs/tutorials/02_converters_for_quadratic_programs.ipynb).